### ISSUE TITLE
How can I tell if someone saved me as contact ?

### ISSUE THREAD

**Issue by fire17** [open-wa/wa-automate-nodejs#1756]
Hey guys, I wish to send someone a message automatically after they've added me as contact
for example, sending my own contact in a group, then checking over the participants if they've added me

I wondering whats the best way to do this...
I've thought about - trying to get their status/lastSeen/profilePic or something that is usually unavailable for non-contacts and then becomes available if they added you. maybe you guys know which is the best marker

I'm wondering if anyone has a good solution to this or maybe there's a function to that which im not aware of, or something else i can use. 
basically i need a function userHasMeAsContact(userID) - i'm alright with it working close to 100% even if its not absolute certainty
Thanks a lot ! :)

**Comment by smashah** [comment]
@fire17 I actually looked into this a while back. There's no way to know if the other account has you saved as a contact.

I will add this to the long term backlog for now
